webapp/src/css/inbox.less

```diff
@@ -1402,14 +1411,45 @@ mm-sidebar-menu .mat-sidenav-container {
       height: calc(@sidebar-menu-height - @sidebar-menu-header-mobile);
     }
 
+    mat-icon {
+      margin-left: -6px !important;
+    }
+
     mm-panel-header {
       height: @sidebar-menu-header-mobile;
+
       .panel-header-title {
         font-size: @font-extra-large;
       }
+
+      .panel-header-close {
+        width: 48px !important;
+      }
+
+      .panel-header-title {
+        margin-left: unset;
+      }
     }
   }
 
+  .app-root:not(.old-nav) .header .tabs a .button-label {
+    /* Keep the desired styles */
+    margin-top: 3px;
+    overflow: hidden;
+    word-wrap: break-word;
+    color: @nav-icon-gray;
+    font-size: @font-XXS;
+    text-overflow: ellipsis;
+
+    /* Unset or override the unwanted properties */
+    display: block;
+    white-space: unset;
+    word-break: unset;
+    -webkit-line-clamp: unset;
+    -webkit-box-orient: unset;
+    line-clamp: unset;
+  }
```

Comment:
Please move this [into L1499](https://github.com/medic/cht-core/blob/167b6fd217b18f91982c7179842c980b6b4bb639/webapp/src/css/inbox.less#L1499), which is pretty much the same selector. 
Then, remove the comments and duplicate attributes.

Reply:
Done.